Sources/Subs-Db-postgresql.php

```diff
@@ -470,6 +441,29 @@ function smf_db_query($identifier, $db_string, $db_values = array(), $connection
 			smf_db_error_backtrace('Hacking attempt...', 'Hacking attempt...' . "\n" . $db_string, E_USER_ERROR, __FILE__, __LINE__);
 	}
 
+	// Debugging.
```

Comment:
I would like to see this code below "// Set optimize stuff" block,
idea would be that the code should be than the same as mysql part.

Reply:
The optimization block expects `$db_cache[$db_count]['q']` to already have a value.

Reply:
... But I suppose there is no strong reason why it would need to be left that way.